Assets/UnityIO.cs

```diff
@@ -0,0 +1,49 @@
+using UnityEngine;
+using System;
+using System.IO;
+using System.Collections;
+
+public delegate void UnityIORequestCallback(object userData, Response response);
+
+public class Response {
```

Comment:
Same here, not sure if a nested class would make more sense, but it is possible to leave `Response` within `UnityIO` scope.